pkg/build/api/types.go

```diff
@@ -58,6 +58,11 @@ type BuildSpec struct {
 
 	// Compute resource requirements to execute the build
 	Resources kapi.ResourceRequirements
+
+	// Optional duration in seconds, counted from the time when a build pod gets
```

Comment:
If this is optional the default behavior is that it does not influence build duration, at least that's my understanding. Why then adding some additional info here.